packages/components/src/card/README.md

```diff
@@ -1,107 +1,105 @@
 # Card
 
-Card provides a flexible and extensible content container.
+<div class="callout callout-alert">
+This feature is still experimental. “Experimental” means this is an early implementation subject to drastic and breaking changes.
+</div>
```

Comment:
`Card` actually isn't experimental so we can remove this notice.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/32243/commits/9ba2635e1dc56e6a6e0aab3c9ec8e551e1be3f5d